<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCS2 Relay Control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon/relay-icon.png</normaloff>assets/icon/relay-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>Target Control</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="mainTab">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabRelay">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="locale">
        <locale language="English" country="Uganda"/>
       </property>
       <attribute name="title">
        <string>Relay control</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>259</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Relay PINS</string>
        </property>
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>51</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="lblNum7">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>51</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="lblNum6">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_7">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>20</y>
           <width>51</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="lblNum5">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb5">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_8">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>100</y>
           <width>51</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNum4">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>100</y>
           <width>51</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNum3">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_9">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>100</y>
           <width>51</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNum2">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>41</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNum1">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>41</width>
           <height>55</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblNum8">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLCDNumber" name="lcdBinState">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>80</y>
          <width>82</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(190, 190, 190);
color: blue</string>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Bin</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>140</y>
          <width>132</width>
          <height>78</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Common modes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="rbNormalMode">
             <property name="text">
              <string>Normal mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbFLashMode">
             <property name="text">
              <string>Flashing mode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);</string>
           </property>
           <property name="text">
            <string>Current state:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtCurrentState">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>232</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabUSBMux">
       <attribute name="title">
        <string>USB Mux control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QGroupBox" name="grbMUXControl">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Multiplexer controler</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QPushButton" name="btnMUXTestingMode">
               <property name="text">
                <string>Acitve Testing / SSH mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMUXFlashMode">
               <property name="text">
                <string>Active Flashing Autosar / Android - QNX mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGraphicsView" name="graphicsViewMUX">
               <property name="enabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPPS">
       <attribute name="title">
        <string>PPS control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Set Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSetChannel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetChannel">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Set Voltage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSetVol">
            <property name="text">
             <string>12</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>V    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetVol">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Get Voltage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtGetVol">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>V    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGetVol">
            <property name="text">
             <string>Get</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Get Ampere</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtGetAmpere">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string>A   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGetAmpere">
            <property name="text">
             <string>Get</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Log:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txtLog"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="menuSetting"/>
    <addaction name="separator"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menuAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
